### PR TITLE
chore: update flopflip

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

Includes flopflip only flushing changes to flag values.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -7163,6 +7163,15 @@ create-jest-runner@^0.5.3:
     jest-worker "^24.0.0"
     throat "^4.1.0"
 
+create-react-class@^15.6.0:
```
Comment (emmenko) [inline yarn.lock]: Hmm why do we get this dependency again? 🤔

Comment (tdeekens) [reply yarn.lock]: I have no idea. Flopflip doesn't use it. I just ran changed the flopflip versions. I would not manually fiddle with the lockfile. Was this causing issues?

Comment (emmenko) [reply yarn.lock]: Just wondering...maybe try regenerating the lockfile.

Comment (tdeekens) [reply yarn.lock]: 566b306

Comment (tdeekens) [reply yarn.lock]: CleanShot 2019-08-30 at 09 49 07@2x

**Comment by tdeekens** [comment]
@emmenko can we merge this?

**Pull request merged**